pkg/api/k8s_versions.go

```diff
@@ -8,6 +8,38 @@ import (
 )
 
 var k8sComponentVersions = map[string]map[string]string{
+	"1.13": {
+		"dockerEngine":                     "1.13.*",
+		"dashboard":                        "kubernetes-dashboard-amd64:v1.10.0",
+		"addon-resizer":                    "addon-resizer:1.8.1",
+		"heapster":                         "heapster-amd64:v1.5.3",
+		"metrics-server":                   "metrics-server-amd64:v0.2.1",
+		"coredns":                          "coredns:1.2.2",
+		"kube-dns":                         "k8s-dns-kube-dns-amd64:1.14.13",
+		"addon-manager":                    "kube-addon-manager-amd64:v8.8",
```

Comment:
This is the only component bumped according to the release notes (besides cluster-proportional-autoscaler, which is still a TODO: for this PR).